mvcc/kvstore_test.go

```diff
@@ -405,10 +405,17 @@ func TestStoreRestore(t *testing.T) {
 	if g := b.tx.Action(); !reflect.DeepEqual(g, wact) {
 		t.Errorf("tx actions = %+v, want %+v", g, wact)
 	}
+
+	gens := []generation{
+		generation{created: revision{4, 0}, ver: 2, revs: []revision{revision{3, 0}, revision{5, 0}}},
+		generation{created: revision{0, 0}, ver: 0, revs: nil},
+	}
+	key1 := []byte{0, 0, 0, 0, 0, 0, 0, 3, 95, 0, 0, 0, 0, 0, 0, 0, 0}
```

Comment:
@xiang90 I had tried printing the values, as even though the values were coming same DeepEqual comparison was failing. Later i tried printing the type and the above is the reason they we failing. Attaching the difference between received value and expected value:
```
Got:
[{Name:insert Params:[g: created[{4 0}] ver[2], revs []mvcc.revision{mvcc.revision{main:3, sub:0}, mvcc.revision{main:5, sub:0}} g: created[{0 0}] ver[0], revs []mvcc.revision(nil)]}], 
Expected:
[{Name:insert Params:[g: created[{4 0}] ver[2], revs []mvcc.revision{mvcc.revision{main:3, sub:0}, mvcc.revision{main:5, sub:0}} g: created[{0 0}] ver[0], revs []mvcc.revision(nil)]}], 
Got:
type : [{%!t(string=insert) [%!t(*mvcc.keyIndex=&{[0 0 0 0 0 0 0 3 95 0 0 0 0 0 0 0 0] {5 0} [{2 {4 0} [{3 0} {5 0}]} {0 {0 0} []}]})]}] 
Expected
type : [{%!t(string=insert) [%!t(*mvcc.keyIndex=&{[102 111 111] {0 0}                        [{2 {4 0} [{3 0} {5 0}]} {0 {0 0} []}]})]}]
```